

arch/ia64/kernel/acpi.c: In function `acpi_map_iosapic':
arch/ia64/kernel/acpi.c:822: error: `MAX_PXM_DOMAINS' undeclared (first use in this function)
arch/ia64/kernel/acpi.c:822: error: (Each undeclared identifier is reported only once
arch/ia64/kernel/acpi.c:822: error: for each function it appears in.)
arch/ia64/kernel/acpi.c:827: error: `pxm_to_nid_map' undeclared (first use in this function)


Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/arch/ia64/kernel/acpi.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN arch/ia64/kernel/acpi.c~ia64-acpi-build-fix arch/ia64/kernel/acpi.c
--- 25/arch/ia64/kernel/acpi.c~ia64-acpi-build-fix	Tue Jan 11 15:36:12 2005
+++ 25-akpm/arch/ia64/kernel/acpi.c	Tue Jan 11 15:38:23 2005
@@ -769,7 +769,7 @@ EXPORT_SYMBOL(acpi_unmap_lsapic);
 #endif /* CONFIG_ACPI_HOTPLUG_CPU */
  
 
-#ifdef CONFIG_NUMA
+#ifdef CONFIG_ACPI_NUMA
 acpi_status __init
 acpi_map_iosapic (acpi_handle handle, u32 depth, void *context, void **ret)
 {
_
